src/DvsaLogger/Factory/DoctrineQueryExtrasFactory.php

```diff
@@ -13,7 +14,6 @@
  */
```

Comment:
i wonder why there was no need for `* @psalm-suppress MissingConstructor`  here

Reply:
I think it is because the class has no properties so Psalm doesn't require a constructor to initialise them